packages/ag-charts-community/src/module/optionModules.ts

```diff
@@ -10,7 +10,26 @@ export interface AxisOptionModule<M extends ModuleInstance = ModuleInstance> ext
 
 export interface SeriesOptionModule<M extends ModuleInstance = ModuleInstance> extends BaseModule {
     type: 'series-option';
-    identifier: string;
+    seriesTypes: readonly (
+        | 'area'
+        | 'bar'
+        | 'box-plot'
+        | 'bubble'
+        | 'heatmap'
+        | 'histogram'
+        | 'line'
+        | 'nightingale'
+        | 'pie'
+        | 'radar-area'
+        | 'radar-line'
+        | 'radial-bar'
+        | 'radial-column'
+        | 'range-area'
+        | 'range-bar'
+        | 'scatter'
+        | 'treemap'
+        | 'waterfall'
+    )[];
```

Comment:
Nit: Let's consider adding a type alias for the series types rather than repeating them?

Perhaps something based on `AgCartesianSeriesOptions['type'] | AgPolarSeriesOptions['type'] | AgHierarchySeriesOptions['type']`?

Reply:
That's a good idea, although for some reason using this type alias triggers the same error as I've mentioned in the comment about `mod.seriesTypes` (but this time the error is with `supportedSeriesTypes`). I'll take a closer look at this on Monday.

Reply:
Done. Added `AgSeriesType` in _packages/ag-charts-community/src/options/series/seriesOptions.ts_